mmdet/models/task_modules/assigners/atss_assigner.py

```diff
@@ -212,23 +219,38 @@ def assign(
         is_pos = candidate_overlaps >= overlaps_thr_per_gt[None, :]
 
         # limit the positive sample's center in gt
-        for gt_idx in range(num_gt):
-            candidate_idxs[:, gt_idx] += gt_idx * num_priors
-        priors_cx = (priors[:, 0] + priors[:, 2]) / 2.0
-        priors_cy = (priors[:, 1] + priors[:, 3]) / 2.0
-        ep_priors_cx = priors_cx.view(1, -1).expand(
-            num_gt, num_priors).contiguous().view(-1)
-        ep_priors_cy = priors_cy.view(1, -1).expand(
-            num_gt, num_priors).contiguous().view(-1)
-        candidate_idxs = candidate_idxs.view(-1)
-
-        # calculate the left, top, right, bottom distance between positive
-        # prior center and gt side
-        l_ = ep_priors_cx[candidate_idxs].view(-1, num_gt) - gt_bboxes[:, 0]
-        t_ = ep_priors_cy[candidate_idxs].view(-1, num_gt) - gt_bboxes[:, 1]
-        r_ = gt_bboxes[:, 2] - ep_priors_cx[candidate_idxs].view(-1, num_gt)
-        b_ = gt_bboxes[:, 3] - ep_priors_cy[candidate_idxs].view(-1, num_gt)
-        is_in_gts = torch.stack([l_, t_, r_, b_], dim=1).min(dim=1)[0] > 0.01
+
+        if isinstance(priors, BaseBoxes):
+            priors_points = priors.centers
+            inside_flag = gt_bboxes.find_inside_points(priors_points)
```

Comment:
need to modify `find_inside_points` function. Add a argument `eps` to make sure the points are inside not on the boudary.